README.rst

```diff
@@ -116,9 +116,9 @@ Edit your buildout.cfg and add add the following to it:
     [versions]
     ...
     collective.js.bootstrap = 2.3.1.1
-    plone.app.blocks = 4.0.6
+    plone.app.blocks = 4.1.0
```

Comment:
Is there anyway that you can add a reference here in the docs to https://github.com/collective/collective.cover/blob/ae9eebdb1fe00b3f829a179992bd889df4ffd0e0/versions-4.3.x.cfg#L16 or would it be more complicated?